requirements.txt

```diff
@@ -28,3 +28,9 @@ uritemplate==3.0.0
 urllib3==1.23
 whitenoise==4.0
 django-rest-swagger==2.2.0
+python-social-auth==0.3.6
+six==1.10.0
+social-auth-app-django==2.1.0
+social-auth-core==1.7.0
+whitenoise==4.0
```

Comment:
whitenose and swagger are already present on lines 29 and 30 respectively.

Reply:
Let me ammend that

Reply:
@parseendavid All done